src/generator-attributes.cs

```diff
@@ -1012,8 +1019,7 @@ public override string ToString ()
 			GenerateSupported (builder);
 			break;
 		case AvailabilityKind.Deprecated:
-			GenerateAdvice (builder);
-			GenerateUnsupported (builder);
+			GenerateDeprecated (builder);
 			break;
 		case AvailabilityKind.Obsoleted:
 			GenerateObsolete (builder);
```

Comment:
I'm sorry about this, but I just remembered that the `UnsupportedOSPlatform` attribute now has a `Message` property we can use to give more information, which means we don't have to use the `Obsolete` attribute for this purpose, and the call to `GenerateObsolete` can be removed as well.

Reply:
Done I believe.